II.pass-tests/03-sum-numbers/add-numbers.js

```diff
@@ -1,4 +1,16 @@
 function addNumbers(numbers) {
-
+/*    var total=0;
+for (i=0; i<numbers.length; i++){
+    total = total + numbers[i];
+}
+return total;
+*/
+total=0;
```

Comment:
Remember to declare your variables with `var`!

Reply:
Thanks. I fixed it.